src/localconf.js

```diff
@@ -285,7 +285,9 @@ function buildConfig(opts) {
   if (!studyRegistrationConfig.studyRegistrationUIDField) {
     studyRegistrationConfig.studyRegistrationUIDField = 'appl_id'
   }
-
+  if (!studyRegistrationConfig.studyRegistrationFormDisclaimerField) {
+    studyRegistrationConfig.studyRegistrationFormDisclaimerField = null
+  }
```

Comment:
we don't really need this and leaves the default value to `undefined`

sorry for not being precise enough when making comments in the ticket